front/src/js/room_list.js

```diff
@@ -4,7 +4,7 @@ const roomList = {
     emits: ["roomSelected", "roomQuit"],
     template: `
     <div class='room_list_wrapper'>
-        <template v-for="(room, index) in rooms" :key="room._id.$oid">
+        <template v-for="(room, index) in rooms" :key="room.id">
             <div class="room_list_item">
                 <div class="room_list_room" @click="$emit('roomSelected', {room: room, index: index})"> {{room.title}} </div>
                 <div class="room_list_quit" @click="$emit('roomQuit', {room: room, index: index})"><div></div></div>
```

Comment:
as stated in index.html, no need to send the index with the event, so the object with both here should just become the room